ta/tpm_log_test/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright (c) 2020, ARM Limited. All rights reserved.
+ */
+
+#ifndef USER_TA_HEADER_DEFINES_H
+#define USER_TA_HEADER_DEFINES_H
+
+#include <ta_tpm_log.h>
+
+#define				TA_UUID TA_TPM_LOG_TEST_UUID
+
+#define TA_FLAGS		TA_FLAG_EXEC_DDR
```

Comment:
`TA_FLAG_EXEC_DDR` refers to a deprecated flag. Prefer simply `#define TA_FLAGS	0`.